requirements/checkdocs.in

```diff
@@ -1,3 +1,5 @@
+pip-tools
+pip-sync-faster
```

Comment:
It looks like when I added `make requirements` to `h` I forgot to add `pip-tools` to all the requirements files. It's supposed to be here, and all the other apps have it, so adding it now